README(site plan).md

```diff
@@ -0,0 +1,10 @@
+Overview (b8-8.5 audis)
+    The practical useage of this website would be for one place for all of audi parts aftermaket and OEM for the cars. it would be a static site that will show the lowest price that a peticular brand is offering 
```

Comment:
How will you keep the listing of prices up to date? Since this site is static, this would be out of scope for the project. I was hoping for a little more detail in the overview. Things to address would be things like the following ...
* Summary of target audience
* Is the site new or are you updating an existing site. I understand from the language used it is new, but it should be stated more clearly.
* The main goal was not clear to me in this objective. Is this a site mainly to provide information on Audi self-service and general part information, or is this a price search engine?